test/test_posv.cc

```diff
@@ -477,17 +385,17 @@ void test_posv_work(Params& params, bool run)
 
             if (verbose > 2) {
                 if (origin == slate::Origin::ScaLAPACK) {
-                    slate::Debug::diffLapackMatrices<scalar_t>(n, n, &A_data[0], lldA, &Aref_data[0], lldA, nb, nb);
+                    slate::Debug::diffLapackMatrices<scalar_t>(n, n, &A_data[0], A_alloc.lld, &Aref_data[0], A_alloc.lld, nb, nb);
                     if (params.routine != "potrf") {
-                        slate::Debug::diffLapackMatrices<scalar_t>(n, nrhs, &B_data[0], lldB, &Bref_data[0], lldB, nb, nb);
+                        slate::Debug::diffLapackMatrices<scalar_t>(n, nrhs, &B_data[0], A_alloc.lld, &Bref_data[0], A_alloc.lld, nb, nb);
```

Comment:
Should this be `B_alloc.lld`? Though I suspect they are equal, since both have `n` rows.

Reply:
I updated this.

Reply:
Yes, you're correct.